egs/tedlium/s5_r2/conf/segment_long_utts.conf

```diff
@@ -0,0 +1,28 @@
+# TF-IDF similarity search options
+max_words=1000
```

Comment:
Its still not sure which of these options are important for various setups. This config can be removed if there are only a few options that are specific to setups.